action.yml

```diff
@@ -69,6 +69,6 @@ inputs:
       converted to HTTPS.
     default: false
 runs:
-  using: node12
+  using: node16

```

Comment:
are we concerned about older GHES + GitHub Connect?

Reply:
We will major version these changes, so users using GitHub Connect can stay pinned on `@v2` so if they are on a very old GHES version their workflows will still run.

Reply:
also discussed offline. sounds like this is the same plan we're using across all in-the-box actions.